--- TerasologyLauncher.bat.orig	2021-05-02 14:29:21.400926009 +0200
+++ TerasologyLauncher.bat	2021-05-02 14:29:22.976979564 +0200
@@ -35,6 +35,9 @@
 @rem Add default JVM options here. You can also use JAVA_OPTS and TERASOLOGY_LAUNCHER_OPTS to pass JVM options to this script.
 set DEFAULT_JVM_OPTS=
 
+@rem Use bundled JRE when available
+if exist %APP_HOME%\jre\ set JAVA_HOME=%APP_HOME%\jre
+
 @rem Find java.exe
 if defined JAVA_HOME goto findJavaFromJavaHome
 
