--- TerasologyLauncher.bat.orig	2020-03-26 13:32:00.000000000 +0100
+++ TerasologyLauncher.bat	2020-03-26 13:33:26.000000000 +0100
@@ -35,6 +35,12 @@
 @rem Add default JVM options here. You can also use JAVA_OPTS and TERASOLOGY_LAUNCHER_OPTS to pass JVM options to this script.
 set DEFAULT_JVM_OPTS=
 
+@rem Use bundled JRE when available
+if not exist %APP_HOME%\jre\ goto findJava
+set JAVA_EXE=%APP_HOME%\jre\bin\java.exe
+goto init
+
+:findJava
 @rem Find java.exe
 if defined JAVA_HOME goto findJavaFromJavaHome
 
@@ -65,6 +71,7 @@
 goto fail
 
 :init
+
 @rem Get command-line arguments, handling Windows variants
 
 if not "%OS%" == "Windows_NT" goto win9xME_args
